src/main/java/seedu/address/model/person/Professor.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.model.person;
+
+import java.util.Objects;
+import java.util.Set;
+
+import seedu.address.model.tag.Tag;
+
+/**
+ * Represents a Professor in the address book.
```

Comment:
change mentions of "address book" to just UniBook

Reply:
Done, all instances of address book are only in .md files